hawtjni-generator/src/main/java/org/fusesource/hawtjni/generator/StructsGenerator.java

```diff
@@ -261,7 +261,7 @@ void generateGetFields(JNIClass clazz) {
         JNIClass superclazz = clazz.getSuperclass();
         String clazzName = clazz.getNativeName();
         String superName = superclazz.getNativeName();
-        String methodname;
+        String Methodname;
```

Comment:
The usual java naming is `methodName`, that would be more consistent.

Reply:
we will use the same naming convention